lib/services/post_service.dart

```diff
@@ -1,4 +1,4 @@
-// ignore_for_file: talawa_good_doc_comments, talawa_api_doc
+// ignore_for_file: talawa_good_doc_comments, talawa_api_doc, avoid_dynamic_calls
```

Comment:
Don't Avoid method calls or property accesses on a "dynamic" target

Reply:
Sir on removing this it is giving avoid dynamic call suggestion

Reply:
So avoid it instead of adding an ignore statement. 

Adding ignore statements for solvable issues is not good practice

Reply:
Done fixing it.